package.json

```diff
@@ -26,9 +26,9 @@
     "devDependencies": {
         "@types/debug": "^4.1.7",
         "@types/node": "^12.20.55",
-        "@types/node-fetch": "^2.6.2",
+        "@types/node-fetch": "2.6.2",
         "all-contributors-cli": "^6.24.0",
-        "deno2node": "^1.7.2"
+        "deno2node": "^1.8.1"
```

Comment:
I guess the version needs to be updated in deno.jsonc backport task as well?

Reply:
Yes